autopilot/contractor.go

```diff
@@ -499,20 +510,20 @@ func (c *contractor) initialContractFunding(settings rhpv2.HostSettings, txnFee,
 	return funding
 }
 
-func (c *contractor) renewFundingEstimate(cfg Config, id types.FileContractID) (types.Currency, error) {
+func (c *contractor) renewFundingEstimate(cfg Config, toRenew rhpv2.ContractRevision) (types.Currency, error) {
 	// fetch contract
-	contract, err := c.ap.bus.Contract(id)
+	contract, err := c.ap.bus.Contract(toRenew.ID())
```

Comment:
doesn't this essentially make `renewalCandidate` moot? In #42 I have `runContractChecks` return 3 contract buckets (delete, ignore, renew), isn't that more straightforward then mixing/matching multiple return types?

Reply:
`AddContract` and `RenewContract` could return the contract they added/renewed, so that saves the fetch after insert, which I think is what you were getting rid of with the `renewalCandidate` in the first place?